### ISSUE TITLE
[build]: **neonkube-build.ps1** build script needs to explicitly  clean

### ISSUE THREAD

**Issue by jefflill** [nforgeio/neonKUBE#455]
**neonkube-build.ps1** build script needs to explicitly remove the **bin** and **obj** folders in the repo before starting the build.  Unfortunately, doing a clean build doesn't actually clean everything.

I encountered a situation where a **project.assets.json** file left over from a previous build (and perhaps a previous version of Visual Studio) was causing the build to fail with a strange error.

**Comment by jefflill** [comment]
I added the **neon-build clean** command to do this but there's not an obvious place to call this in the build script due to an chicken-and-egg type issue.  The problem is that the tool needs to be built too.

I'm putting this on the backlog.  I believe that we saw the issue above because I did the build on my Fremont workstation where I probably hadn't done a build since upgrading to the Visual Studio 2019 preview.  So I think it takes a major VS upgrade to break this.

I'm putting this on the backlog for the future.
